config/dev.exs

```diff
@@ -44,7 +44,7 @@ config :remote_retro, RemoteRetro.Mailer, adapter: Bamboo.LocalAdapter
 
 config :remote_retro, :oauth_client, OAuth2.Client
 
-config :remote_retro, :add_mock_user_to_session, true
+config :remote_retro, :add_mock_user_to_session, false
```

Comment:
I believe this is the only place this config is referenced in the app, so we can delete this line outright